.github/workflows/actions_build.yml

```diff
@@ -0,0 +1,145 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  build_on_linux:
+    runs-on: ubuntu-latest
```

Comment:
The default timeout of a job is 6 hours. It seems to take too long to detect a hanging build.
How about setting `timeout-minutes: 60`?

Reply:
That's a good suggestion. 👍